lib/mix/tasks/phx.gen.cert.ex

```diff
@@ -270,6 +259,19 @@ defmodule Mix.Tasks.Phx.Gen.Cert do
     )
   end
 
+  defp cert_date(date, days_to_add \\ 0) do
+    {year, month, day} =
+      date
+      |> Date.add(days_to_add)
+      |> Date.to_erl()
+
+    yy = year |> Integer.to_string() |> String.slice(2, 2)
+    mm = month |> Integer.to_string() |> String.pad_leading(2, "0")
+    dd = day |> Integer.to_string() |> String.pad_leading(2, "0")
+
+    yy <> mm <> dd
```

Comment:
@quitejonny you can also use Date.to_iso8601(date, :basic) which does all of the above. The I don’t think we need the helper function. :) could you send another patch?

Reply:
I opened a new PR #5738 which uses the Date.to_iso8601 function instead of a custom implementation.